python/ray/tune/schedulers/pbt.py

```diff
@@ -184,9 +199,11 @@ def __init__(self,
                  custom_explore_fn=None,
                  log_config=True):
         for value in hyperparam_mutations.values():
-            if not (isinstance(value, (list, dict)) or callable(value)):
+            if not (isinstance(value,
+                               (list, dict, sample_from)) or callable(value)):
                 raise TypeError("`hyperparam_mutation` values must be either "
-                                "a List, Dict, or callable.")
+                                "a List, Dict, a tune.sample_from instance or "
```

Comment:
`sample_from` is not exposed to the user, so you'll have to say something like "a tune search space object".

Reply:
done